Sources/Plasma/FeatureLib/pfPython/pyGUIControlListBox.cpp

```diff
@@ -610,27 +610,25 @@ int16_t   pyGUIControlListBox::AddImage( pyImage& image, bool respectAlpha )
 }
 
 
-int16_t pyGUIControlListBox::FindString( const ST::string &toCompareTo )
+int16_t pyGUIControlListBox::FindString( ST::string toCompareTo )
 {
     if ( fGCkey )
     {
         // get the pointer to the modifier
         pfGUIListBoxMod* plbmod = pfGUIListBoxMod::ConvertNoRef(fGCkey->ObjectIsLoaded());
         if ( plbmod )
-            return plbmod->FindString(toCompareTo);
+            return plbmod->FindString(std::move(toCompareTo));
     }
     return -1;
 }
 
-int16_t pyGUIControlListBox::AddTextWColor( const char *str, pyColor& textcolor, uint32_t inheritalpha)
+int16_t pyGUIControlListBox::AddTextWColor( ST::string str, pyColor& textcolor, uint32_t inheritalpha)
 {
-    wchar_t *wStr = hsStringToWString(str);
-    int16_t retVal = AddTextWColorW(wStr,textcolor,inheritalpha);
-    delete [] wStr;
+    int16_t retVal = AddTextWColorW(std::move(str), textcolor, inheritalpha);
     return retVal;
 }
 
-int16_t pyGUIControlListBox::AddTextWColorW( const std::wstring& str, pyColor& textcolor, uint32_t inheritalpha)
+int16_t pyGUIControlListBox::AddTextWColorW( ST::string str, pyColor& textcolor, uint32_t inheritalpha)
```

Comment:
line 639
```C++
pfColorListElement *element = new pfColorListElement(str.c_str(), textcolor.getColor(), nullptr, hsColorRGBA(), inheritalpha);
```

Is sending UTF-8 into a function that reencodes a `const char*` (assumed to be latin_1) as UTF-16. You'll want to do `to_wchar()` there or `ST::string`-size `pfColorListElement`.